ci/build_container/build_recipes/boringssl.sh

```diff
@@ -1,10 +1,16 @@
+#!/bin/bash
+
 set -e
 
+COMMIT=be2ee342d3781ddb954f91f8a7e660c6f59e87e5
+
 git clone https://boringssl.googlesource.com/boringssl
 cd boringssl
-git reset --hard be2ee342d3781ddb954f91f8a7e660c6f59e87e5
+git reset --hard $COMMIT
 cmake .
 make
 cp -r include/* $THIRDPARTY_BUILD/include
 cp ssl/libssl.a $THIRDPARTY_BUILD/lib
 cp crypto/libcrypto.a $THIRDPARTY_BUILD/lib
+cd ..
+rm -rf boringssl
```

Comment:
I don't think this is necessary, this is handled by the recursive make file that wraps build recipes. See https://github.com/lyft/envoy/blob/master/ci/build_container/Makefile#L26. I'd suggest remove this from the various recipes, as it's just clutter.